packages/cosmic-swingset/lib/launch-chain.js

```diff
@@ -153,8 +153,16 @@ export async function launch(
     schedulerBlockTimeHistogram.record((now - blockStart) / 1000);
   }
 
+  let firstBlock;
   async function endBlock(_blockHeight, _blockTime) {
-    await crankScheduler(FIXME_MAX_CRANKS_PER_BLOCK);
+    let numCranks = FIXME_MAX_CRANKS_PER_BLOCK;
+    if (firstBlock) {
```

Comment:
My habit is to look for a test for any fix. But I guess unit testing this stuff hasn't been cost-effective to date.

Reply:
Yeah, the test really needs to exercise the Golang code as part of the chain and see that the first boot of the chain can produce blocks beyond 2 in order to discover the absence of the problem it fixes.  Expensive, unfortunately.